snyk/models.py

```diff
@@ -24,7 +24,7 @@ class Vulnerability(DataClassJSONMixin):
     isPatchable: bool
     identifiers: Any
     semver: Any
-    fromPackages: List[str]
+    fromPackages: Optional[List[str]] = None
```

Comment:
In general we've been setting default values for list optional fields to a list rather than none. If you could change to:

```python
Optional[List[str]] = field(default_factory=list)
```

Reply:
Updated